index.html

```diff
@@ -69,7 +69,7 @@
 
                 <div class="outside-in-headerTitle">
                         <span class="outside-in-headerTitle-icon"></span>
-                        <h1>OUTSIDE·IN</h1>
+                        <h1>OUTSIDEIN</h1>
```

Comment:
Was the removal of the 'dash' between 'outside' and 'in' intentional, @dayotte? The dash is still present everywhere else...